third-party/realdds/include/realdds/dds-stream.h

```diff
@@ -0,0 +1,92 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+
+#include <functional>
+#include <memory>
+#include <string>
+
+namespace realdds {
+
+template<typename T, typename P>
+inline bool is( std::shared_ptr<P> ptr )
```

Comment:
I don't think these should be in this header... I'd say they don't even belong in realdds

Reply:
Removed from header, and removed the check from the cpp, knowing that in your upcoming PR video/motion profile objects will be changed anyway